opentelemetry-api/src/opentelemetry/metrics/time_series.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+
+
+class CounterTimeSeries:
```

Comment:
I guess I have the same question regarding the `TimeSeries` name :)

Reply:
In https://github.com/open-telemetry/oteps/pull/29 this becomes `Handle`. Please review.